smithy-aws-protocol-tests/model/ec2-query/empty-input-output.smithy

```diff
@@ -3,8 +3,9 @@
 
 $version: "0.5.0"
 
-namespace aws.protocols.tests.ec2
+namespace aws.protocoltests.ec2
```

Comment:
I renamed these tests to take a page from JPMS modules about split packages. It probably doesn't matter nor ever will, but just in case...